ranger-zookeeper/src/test/java/com/flipkart/ranger/zookeeper/serviceprovider/BaseServiceProviderBuilderTest.java

```diff
@@ -95,5 +95,6 @@ public void testbuilder() throws Exception {
                 .withPort(port)
                 .withHealthUpdateIntervalMs(1000)
                 .build();
+        serviceProvider.start();
```

Comment:
maybe assert something here?

Reply:
Idea is to see if provider is starting up without exceptions, have added not-null assertion for completion.